### ISSUE TITLE
Crash when using decimal values in file limit size

### ISSUE THREAD

**Issue by Black-Light** [nixxquality/WebMConverter#120]
When attempting to output with the file limit set to 8.2 for instance I received this error

```
Arguments: -f nut -i pipe:0   -c:v libvpx -pix_fmt yuv420p -threads 8 -slices 2 -metadata title="*****" -ac 2 -c:a libvorbis -minrate:v 287K -b:v 287K -maxrate:v 287K -bufsize 172K -rc_init_occupancy 1148K -qcomp 1 -fs 8.2M -b:a 64K -f webm -y "********.webm"
--- CREATING AVISYNTH PROXY --- 
Guessed Channel Layout for  Input Stream #0.1 : stereo
Input #0, nut, from 'pipe:0':
  Metadata:
    encoder         : Lavf55.44.100
--- FFMPEG HAS EXITED ---
  Duration: N/A, start: 0.000000, bitrate: 3072 kb/s
    Stream #0:0: Video: rawvideo (I420 / 0x30323449), yuv420p, 1024x768, 30 fps, 30 tbr, 61440 tbn, 61440 tbc
    Stream #0:1: Audio: pcm_f32le (PFD  / 0x20444650), 48000 Hz, stereo, flt, 3072 kb/s
Expected int64 for fs but found 8.2M

ffmpeg.exe exited with exit code 1. That's usually bad.
```

After closing this window the program crashes.

**Comment by nixxquality** [comment]
Seems to be an issue with ffmpeg. I tried using a newer version but that didn't fix it.

Not sure why it happens, either. 8200000 should definitely be representable as a 64-bit integer.

**Comment by nixxquality** [comment]
You'll just have to not use the size limit field for now. If you want the automatic bitrate, just fill out the bitrate field with whatever gets automatically generated, then empty the size limit field.

I'll make a proper workaround in a while.

**Comment by nixxquality** [comment]
Included in [v2.17.0](https://github.com/nixxquality/WebMConverter/releases/tag/v2.17.0).
